Makefile

```diff
@@ -84,3 +84,143 @@ ci-test:
 		done; \
 	fi; \
 	}
+
+.PHONY: stack-build
```

Comment:
@power-fungus i don't think we need dual targets (`stack` and `nix` based). unless you disagree, it might be better to adapt the current targets to use `stack` instead of adding additional targets. i foresee the following `make` target changes:
- `default.nix` gets removed
- `build`/`init` gets updated to use `stack`
- `test`, `bench`, `doc`, `package` gets updated to use `stack`
- `ci-test` target gets updated to invoke appropriate parameterized `stack` command (in place of parameterized `nix` command)
- `devel`, `tags`, `test-all` remain unaltered

does the above sound about right?

Reply:
It does sound good, but I have no idea about your workflow and your requirements. Personally I only use stack for development. Plus: I have no idea if I will get functional equivalence between the current and the stack-based commands

Reply:
Should be implemented now, but with the constraint that benchmark and docs are made to a location where stack feels like it. This is printed to the console after the commands. If this is unacceptable, I will have to take a more detailed look in how to fix this.